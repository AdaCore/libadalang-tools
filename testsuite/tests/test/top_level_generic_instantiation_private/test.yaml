description:
  Check that a warning is emitted when a top level generic package contains
  private subprograms.

driver: shell_script
