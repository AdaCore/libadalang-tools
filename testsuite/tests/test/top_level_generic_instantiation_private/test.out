warning (TGen): generic package Bar with private declarations is not supported.
bar.ads:4:4: info: corresponding test PASSED
bar.ads:4:4: info: corresponding test PASSED
bar.ads:4:4: info: corresponding test PASSED
bar.ads:4:4: info: corresponding test PASSED
bar.ads:4:4: info: corresponding test PASSED
5 tests run: 5 passed; 0 failed; 0 crashed.
